libraries/standard/mqtt/test/access/iot_test_access_mqtt_api.c

```diff
@@ -53,7 +53,7 @@ _mqttConnection_t * IotTestMqtt_createMqttConnection( bool awsIotMqttMode,
 
 void IotTestMqtt_mqttOperation_tryDestroy( void * pData )
 {
-    return _mqttOperation_tryDestroy( pData );
+    _mqttOperation_tryDestroy( pData );
```

Comment:
Not a blocker, but the verb 'try' here isn't reassuring.  How do I know if it did anything if it doesn't return a status?

Reply:
It sets a flag in the parameter to let other users know that the subscription couldn't be destroyed.